lib/pure/osproc.nim

```diff
@@ -158,6 +158,16 @@ proc processID*(p: Process): int {.rtl, extern: "nosp$1".} =
   ## returns `p`'s process ID.
   return p.id
 
+proc processID*(): int =
+  ## return current process ID
+  when defined(windows):
+    type DWORD = uint32
+    proc GetCurrentProcessId(): DWORD {.stdcall, dynlib: "kernel32",
```

Comment:
```suggestion
    proc GetCurrentProcessId(): uint {.stdcall, dynlib: "kernel32",
```

Reply:
uint32 in 32 bit target
uint64 in 64 bit target

Reply:
are you sure? 

see
* https://stackoverflow.com/questions/39419/how-large-is-a-dword-with-32-and-64-bit-code
* https://msdn.microsoft.com/en-us/library/cc230318.aspx

Reply:
I'm not sure. :-)
Hmm `DWORD* = int32` in winlean.nim
Why?!

Reply:
good catch, filed https://github.com/nim-lang/Nim/issues/9848